src/FindApprenticeshipTraining/SFA.DAS.FindApprenticeshipTraining/Application/Courses/Queries/GetCourseByLarsCode/GetCourseByLarsCodeQueryHandler.cs

```diff
@@ -36,15 +40,17 @@ public async Task<GetCourseByLarsCodeQueryResult> Handle(GetCourseByLarsCodeQuer
 
         ApprenticeshipFunding apprenticeshipFunding = standardDetails.ApprenticeshipFunding?.Count > 0 ?
             standardDetails.ApprenticeshipFunding[standardDetails.ApprenticeshipFunding.Count - 1] :
-            null;
+        null;
+
+        LocationItem locationItem = await _cachedLocationLookupService.GetCachedLocationInformation(query.Location, 0, 0);
 
         var courseTrainingProvidersCountResponse =
             await _roatpCourseManagementApiClient.GetWithResponseCode<GetCourseTrainingProvidersCountResponse>(
                 new GetCourseTrainingProvidersCountRequest(
                     [query.LarsCode],
                     query.Distance,
-                    query.Lat,
-                    query.Lon
+                    (decimal?)locationItem?.GeoPoint?[0] ?? null,
+                    (decimal?)locationItem?.GeoPoint?[1] ?? null
```

Comment:
We will have to interpret these in many places, perhaps we should have these as readonly properties on the LocationItem so we do not have to repeat this calculations in each place.